build/index.html

```diff
@@ -2257,6 +2257,233 @@ <h5 class="head-5 text--white margin-bottom-medium">Delete a Video</h5>
   </tbody>
 </table>
 
+
+<!-- ___LIVE_EVENTS____________________________ -->
+
+<h2 class="head-3 margin-top-xlarge padding-top-xlarge border-top margin-bottom-medium" id="live-events">Live Events</h2>
+
+<section class="text-2 contain">
+  <p>A live event represents a Vimeo event on Vimeo.com.</p>
```

Comment:
Have we been referring to these as "Vimeo Event" or "Vimeo event"? 🤔 Just want to confirm that we are keeping the capitalization the same throughout, particularly for user-facing APIs/interfaces.

Reply:
Good catch, looks like we've been calling them "Vimeo Event" in the admin and the [Vimeo API docs](https://developer.vimeo.com/api/live/events) do the same. I'll update it!

Reply:
There's another spot in the docs where I was inconsistent with the capitalization, so I pushed a commit to fix that too ✏️